db/migrations/20220512132042_create_repository_configurations_table.up.sql

```diff
@@ -1,10 +1,10 @@
 BEGIN;
 
 CREATE TABLE IF NOT EXISTS repository_configurations(
-    uuid VARCHAR (255) UNIQUE NOT NULL PRIMARY KEY,
+    uuid UUID UNIQUE NOT NULL PRIMARY KEY,
```

Comment:
note i chnaged this to use the UUID type in postgresql, it may have some slight performance advantages